el/src/main/java/module-info.java

```diff
@@ -0,0 +1,7 @@
+module jakarta.cdi.el {
+    exports jakarta.enterprise.inject.spi.el;
+
+    requires transitive jakarta.cdi;
+
+    requires static jakarta.el;
```

Comment:
Requires static is the correct dependency since the imported type is used in API. One should be able to only require the jakarta.cdi.el module and have access to the jakarta.el module.

Reply:
If you want consumers of `jakarta.cdi.el` to also have immediate access to `jakarta.el`, then it should be `requires transitive` IIUC?

Reply:
Ah, that's what you said. OK, then I agree with you :-)

Will amend.

Reply:
Done. Thank you!